### DISCUSSION TITLE
Looking for information on building an outdoor enclosure

### DISCUSSION THREAD

**Discussion by nbanderso** [mcguirepr89/BirdNET-Pi#781] [General]
Hi all,

I have been limping along with BirdnetPi by leaving my pi semi-protected under an overhang of my patio.  While this has worked thus far, I am looking to build something that is a little more permanent.  

More specifically, I want to put together a waterproof enclosure that allows for proper cooling, has power supply, and can hold multiple raspberry pi’s. 

I am a bit of a novice when it comes to electronics so this is definitely uncharted waters for me.  Does anyone know of any good guides that can get me going in the right direction?  

Thanks in advance!

**Comment by ampsu** [comment]
I guess that waterproof junction/electrical boxes are one way to go. If you can get away with one Pi then there are some waterproof ready to go cases: https://uk.pi-supply.com/products/die-cast-outdoor-weatherproof-enclosure and https://kksb-cases.com/products/kksb-waterproof-sbc-case

**Reply by becausebirds** [reply]
I built an outdoor enclosure for my Pi. I detailed it here: [https://becausebirds.com/fun-raspberry-pi-project-identify-backyard-birds-24-7/](https://becausebirds.com/fun-raspberry-pi-project-identify-backyard-birds-24-7/)

Hopefully this helps!

**Reply by nbanderso** [reply]
@becausebirds this is exactly what I was looking for.  Thank you!

Can you expand on the wiring a bit more?  What is happening on the left USB connector on the enclosure?  Is that feeding the Pi with power?  If so, how is to converting a traditional AC plug to USB?  

Is the right USB connector going to the Microphone?  How is the USB soundcard encorporated into that?

Thanks again.

**Reply by becausebirds** [reply]
Yep! The two holes on the bottom are waterproof USB ports, the right one is facing outward, and the USB microphone plugs into it. The left one is reversed and has the USB port on the inside of the case. The one on the left has a USB-A to micro-USB cable plugged into it going into the Pi's power port. 

Each of these ports on the bottom is USB-A Male to Female. The left one is plugged into a power brick that's plugged into the outlet next to the box. 

If you use a USB microphone, the soundcard isn't needed. If you are using a 3.5mm microphone, you'll need the soundcard dongle thing. Hope this helps! I listed all the parts I used in the article at the top.

**Comment by lloydbayley** [comment]
There may be various degrees of outrage to this comment, however, water can wick in almost anywhere. To this end, I always have a very small hole at the bottom of an enclosure (even if I have waterproofed most of it) to let out any condensation or moisture that might accumulate. When I say small hole, I mean small....1/16" is ok. Not to let bugs in (some may make it), if you're worried, a fine piece of mesh over the hole will hopefully stop them.
